app/models/work.rb

```diff
@@ -2,8 +2,8 @@
 # frozen_string_literal: true
 
 class Work < ApplicationRecord
-  has_many :contributors
-  has_many :related_links
-  has_many :related_works
+  has_many :contributors, dependent: :nullify
```

Comment:
@jcoyne are these the right dependent actions for these three relations? I took a big guess, so pls let me know.

Reply:
I'm thinking `:destroy` might be better.

Reply:
done